azurerm/internal/services/datafactory/data_factory_trigger_blob_event_resource_test.go

```diff
@@ -90,6 +90,30 @@ func TestAccDataFactoryTriggerBlobEvent_update(t *testing.T) {
 	})
 }
 
+func TestAccDataFactoryTriggerBlobEvent_startStop(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_data_factory_trigger_blob_event", "test")
+	r := TriggerBlobEventResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.activated(data, true),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("activated").HasValue("true"),
+			),
+		},
+		data.ImportStep(),
+		{
+			Config: r.activated(data, false),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("activated").HasValue("false"),
+			),
+		},
+		data.ImportStep(),
```

Comment:
might be overkill but should we check we can update this back to true too?

Reply:
Sure, thanks!